x11-wm/pekwm/pekwm-0.2.1.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+CMAKE_ECLASS=cmake
+inherit cmake-multilib desktop
+
+DESCRIPTION="A lightweight window manager initially based on aewm++"
+HOMEPAGE="
+	https://www.pekwm.se/
+	https://github.com/pekdon/pekwm
+"
+SRC_URI="
+	https://github.com/pekdon/${PN}/releases/download/release-${PV}/pekwm-${PV}.tar.gz
+"
+S="${WORKDIR}"/${PN}\-${PV}
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="contrib debug +jpeg +png themes truetype +xft xinerama +xpm"
+
+RDEPEND="
+	png? ( media-libs/libpng:0 )
+	jpeg? ( virtual/jpeg:0 )
+	xpm? ( x11-libs/libXpm )
+	x11-libs/libXrender
+	truetype? ( x11-libs/libXft )
+	xinerama? ( x11-libs/libXinerama )
+	themes? ( dev-vcs/git )"
+
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DENABLE_XSHAPE="yes"
+		-DENABLE_XRANDR="yes"
+		-DENABLE_JPEG=$(usex jpeg)
+		-DENABLE_PNG=$(usex png)
+		-DENABLE_XINERAMA=$(usex xinerama)
+		-DENABLE_XFT=$(usex truetype)
+		-DENABLE_XPM=$(usex xpm)
```

Comment:
Some of these seem to be undefined, 
```
* One or more CMake variables were not used by the project:
 *   ENABLE_JPEG
 *   ENABLE_PNG
 *   ENABLE_XPM
 *   ENABLE_XRANDR
 *   ENABLE_XSHAPE
```

Reply:
I'm not expert on CMakeLists.txt, but those are defined in the pekwm CMakeLists.txt (and all of them are "ON" by default). I didn't notice this before, but yes this warning is present. My first guess is that they are defined in CMakeLists.txt and not used then, because the definition in the CMakeLists.txt is the same for ENABLE_XINERAMA, but it doesn't complain about this variable. I'll try to figure it out.

Reply:
My fault, they should be ENABLE_IMAGE_[JPEG,PNG,XPM] instead of ENABLE_[JPEG,PNG,XPM]. And ENABLE_XSHAPE should be ENABLE_SHAPE, the same for ENABLE_XRANDR -> ENABLE_RANDR. And ENABLE_SHAPE and ENABLE_RANDR is "ON" in CMakeLists.txt, so don't have to be listed. Corrected fd21711.